### PR TITLE
Configure for use with RS256, remove userinfo audience

### PR DESCRIPTION
**Pull request by jimmyjames**

### CODE REVIEW

**Comment by jimmyjames** [comment]
Quickstart article PR that accompanies this change: https://github.com/auth0/docs/pull/8443

File: 01-Login/README.md [hunk 1]
```diff
@@ -3,17 +3,22 @@
...
 ### Auth0 Dashboard
-1. On the [Auth0 Dashboard](https://manage.auth0.com/#/clients) create a new Client of type `Regular Web Application`. 
-1. Add the URL that will be called on an OAuth successful login to the Allowed Callback URLs. i.e.: `https://mysite.com/callback`.
-1. Add the URL that will be called on logout to the Allowed Logout URLs. i.e.: `https://mysite.com/logout`.
-1. Copy the `Domain`, `Client ID` and `Client Secret` values at the top of the page and use them to configure the Java Application.
 
+1. On the [Auth0 Dashboard](https://manage.auth0.com/#/clients), click **CREATE APPLICATION**, provide a name for your Application, select **Regular Web Application**, and click **Create**
+1. Go to the **Settings** tab of your Application
+1. Add the URL `http://localhost:3000/callback` to the **Allowed Callback URLs** field
+1. Add the URL `http://localhost:3000/login` to the **Allowed Logout URLs** field
```
Comment (joshcanhelp) [inline 01-Login/README.md]: `/login` is correct to add to Logout URLs?

File: 01-Login/README.md [hunk 2]
```diff
@@ -3,17 +3,22 @@
...
+1. Click **SAVE CHANGES**
+1. The `Domain`, `Client ID`, and `Client Secret` values will be used next to configure the Java application
 
 ### Java Application
-Set the client values in the `src/main/webapp/WEB-INF/web.xml` file. They are read by the `AuthenticationControllerProvider` class.
+
+Set the client values in the `src/main/webapp/WEB-INF/web.xml` file.
```
Comment (joshcanhelp) [inline 01-Login/README.md]:
```suggestion
Set the Auth0 Application values from above in the `src/main/webapp/WEB-INF/web.xml` file.
```

File: 01-Login/README.md (hunk 1 above)
Comment (jimmyjames) [reply 01-Login/README.md]: It is. The sample could use an overhaul, but not part of this PR.

**Pull request merged**